content/articles/shell-script-to-demonstrate-the-se-of-shell-function-library/index.md

```diff
@@ -1,28 +1,71 @@
+
 ### Introduction
-The Shell Function Library is a set of functions that may be called from any location in the development environment. Shell scripting becomes a little less laborious and repetitious as a result. We may access and call such functions from other files or scripts by building a shell script with some functions specified in it. It aids in the avoidance of code repetition in huge files and sophisticated scripts.
+What are functions? They are blocks of code that carry out a certain activity. Calling it and reusing it is possible. This bring a Shell Function Library which is a set of functions that may be called from any location in the development environment. Shell scripting becomes a little less laborious and repetitious as a result. We may access and call such functions from other files or scripts by building a shell script with some functions specified in it. It aids in the avoidance of code repetition in huge files and sophisticated scripts.
 

```

Comment:
A good computer program has a lot of codes in it. These codes can sometimes be repeated and used multiple times in the program. To simplify the process and reduce errors, functions are used to group pieces of code that are used repeatedly.
 
In shell scripting, functions library may be called from any location in the development environment, thus making shell scripting less laborious and reduces errors.
This article will take you through how to create a simple shell script function and how to use it.